model.py

```diff
@@ -187,6 +258,12 @@ def __init__(
             torch.ones(self.nb_output_bins).float()
         )
 
+    def freeze(self):
+        # set all parameters as not requiring gradient, more RAM-efficient
+        # at test time
+        for p in self.parameters():
+            p.requires_grad = False
```

Comment:
maybe we can also set this [to eval here](https://github.com/PyTorchLightning/pytorch-lightning/blob/caa9c6760b282637c5f2e7408f85685cc3b342cf/pytorch_lightning/core/lightning.py#L1592)

Reply:
sure

Reply:
done